analyzer/src/test/kotlin/managers/GradleDependencyHandlerTest.kt

```diff
@@ -287,6 +295,28 @@ class GradleDependencyHandlerTest : WordSpec({
             scopeDependencies should containExactly(RootDependencyIndex(0))
         }
     }
+
+    "GradleDependencyHandler" should {
+        "handle an exception when resolving a package" {
+            val maven = mockk<MavenSupport>()
+            val exception =
+                ProjectBuildingException("testProject", "Could not build.", IOException("Download exception"))
```

Comment:
Nit: Maybe use just "project" instead of "testProject" to make this fit into a single line?